test/option/stack-size.py

```diff
@@ -89,29 +89,29 @@
 #
 # Test without any options
 #
-test.run(chdir='work1', 
```

Comment:
My editor stripped the whitespace in this file. Let me know if you want me to put it back.

Reply:
No.  Mine are set to do that too, and since it makes checkers that bit quieter, stripped is better.  For a while we had a tool that ran in CI that reported lint-style failures on changes in PRs and that was used to make sure trailing whitespace was not accidentally added. That got discontinued, so it's only manual for now.

Reply:
No problem. github diff you can tell it to ignore whitespace right?

Reply:
Oh, for sure. Some folks just dislike any sort of extraneous changes - interferes with git blame, etc.

Reply:
*cough* @bdbaddog pokes me about that often enough ("you're making my life impossible", etc.)

Reply:
> _cough_ @bdbaddog pokes me about that often enough ("you're making my life impossible", etc.)

:) well to be fair, your code formatting often splits lines, so the ignore whitespace doesn't ignore those.

Reply:
indeed... if we do the black reformat, there'll be somewhat less need for that. adding any typing to function signatures usually makes 'em way too wide, sigh.